src/foam/nanos/dig/drivers/DigFormatDriver.js

```diff
@@ -280,7 +280,8 @@ foam.CLASS({
         // the existing behavior.
         var clientEx = ce.getClientRethrowException();
         if ( clientEx instanceof ValidationException ) {
-          throw new DAOPutException(clientEx.getMessage(), ce);
+          throw new DAOPutException(
+            ((ValidationException) clientEx).getMessage(), ce);
```

Comment:
should this be 'getExceptionMessage' ?? otherwise, i'm not sure why you are casting to a ValidationException.